src/main/resources/environments.json

```diff
@@ -0,0 +1,13 @@
+{
+  "staging": {
+    "baseUrl": "https://jsonplaceholder.typicode.com/"
+  },
+  "preProd": {
+    "baseUrl": "https://jsonplaceholder.typicode.com/"
+  },
+  "common": {
+    "posts": "posts",
+    "comments": "comments",
+    "users": "users"
+  }
```

Comment:
what is it?

Reply:
One solution I tried to follow but I am not going to do it. File removed